tests/sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/test_only_metrics_ping/expect.yaml

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
client-3 doesn't report a baseline nor a metrics ping in this submission_date, but it did report a metrics ping at some point, as seen in _previous (existing table records).
client-2 did not report a baseline ping and doesn't have a record in _previous. It is only reporting a metrics ping.